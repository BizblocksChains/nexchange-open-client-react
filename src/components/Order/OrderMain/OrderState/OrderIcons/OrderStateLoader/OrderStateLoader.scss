.loader {
  margin: 10px 0 -11px 0;

  svg path,
  svg rect {
    fill: $green;
  }
}
