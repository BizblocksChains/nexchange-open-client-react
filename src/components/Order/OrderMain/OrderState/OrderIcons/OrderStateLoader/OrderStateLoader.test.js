import React from 'react';
import { shallow } from 'enzyme';
import OrderStateLoader from './OrderStateLoader';

describe('OrderStateLoader', () => {
  it('renders correctly', () => {
    const wrapShallow = shallow(<OrderStateLoader />);
    expect(wrapShallow).toMatchSnapshot();
  });
});
