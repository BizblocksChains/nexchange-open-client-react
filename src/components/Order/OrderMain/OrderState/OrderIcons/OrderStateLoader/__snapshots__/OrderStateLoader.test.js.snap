// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderStateLoader renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <OrderStateLoader />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <svg
        height="40px"
        id="loader-1"
        style={
                Object {
                        "enableBackground": "new 0 0 50 50",
                      }
        }
        version="1.1"
        viewBox="0 0 50 50"
        width="40px"
        x="0px"
        xmlns="http://www.w3.org/2000/svg"
        y="0px"
>
        <path
                d="M43.935,25.145c0-10.318-8.364-18.683-18.683-18.683c-10.318,0-18.683,8.365-18.683,18.683h4.068c0-8.071,6.543-14.615, 14.615-14.615c8.072,0,14.615,6.543,14.615,14.615H43.935z"
                fill="#000"
        >
                <animateTransform
                        attributeName="transform"
                        attributeType="xml"
                        dur="0.6s"
                        from="0 25 25"
                        repeatCount="indefinite"
                        to="360 25 25"
                        type="rotate"
                />
        </path>
</svg>,
      "className": "loader",
      "title": "2",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <path
          d="M43.935,25.145c0-10.318-8.364-18.683-18.683-18.683c-10.318,0-18.683,8.365-18.683,18.683h4.068c0-8.071,6.543-14.615, 14.615-14.615c8.072,0,14.615,6.543,14.615,14.615H43.935z"
          fill="#000"
>
          <animateTransform
                    attributeName="transform"
                    attributeType="xml"
                    dur="0.6s"
                    from="0 25 25"
                    repeatCount="indefinite"
                    to="360 25 25"
                    type="rotate"
          />
</path>,
        "height": "40px",
        "id": "loader-1",
        "style": Object {
          "enableBackground": "new 0 0 50 50",
        },
        "version": "1.1",
        "viewBox": "0 0 50 50",
        "width": "40px",
        "x": "0px",
        "xmlns": "http://www.w3.org/2000/svg",
        "y": "0px",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <animateTransform
            attributeName="transform"
            attributeType="xml"
            dur="0.6s"
            from="0 25 25"
            repeatCount="indefinite"
            to="360 25 25"
            type="rotate"
/>,
          "d": "M43.935,25.145c0-10.318-8.364-18.683-18.683-18.683c-10.318,0-18.683,8.365-18.683,18.683h4.068c0-8.071,6.543-14.615, 14.615-14.615c8.072,0,14.615,6.543,14.615,14.615H43.935z",
          "fill": "#000",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "attributeName": "transform",
            "attributeType": "xml",
            "dur": "0.6s",
            "from": "0 25 25",
            "repeatCount": "indefinite",
            "to": "360 25 25",
            "type": "rotate",
          },
          "ref": null,
          "rendered": null,
          "type": "animateTransform",
        },
        "type": "path",
      },
      "type": "svg",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <svg
          height="40px"
          id="loader-1"
          style={
                    Object {
                              "enableBackground": "new 0 0 50 50",
                            }
          }
          version="1.1"
          viewBox="0 0 50 50"
          width="40px"
          x="0px"
          xmlns="http://www.w3.org/2000/svg"
          y="0px"
>
          <path
                    d="M43.935,25.145c0-10.318-8.364-18.683-18.683-18.683c-10.318,0-18.683,8.365-18.683,18.683h4.068c0-8.071,6.543-14.615, 14.615-14.615c8.072,0,14.615,6.543,14.615,14.615H43.935z"
                    fill="#000"
          >
                    <animateTransform
                              attributeName="transform"
                              attributeType="xml"
                              dur="0.6s"
                              from="0 25 25"
                              repeatCount="indefinite"
                              to="360 25 25"
                              type="rotate"
                    />
          </path>
</svg>,
        "className": "loader",
        "title": "2",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <path
            d="M43.935,25.145c0-10.318-8.364-18.683-18.683-18.683c-10.318,0-18.683,8.365-18.683,18.683h4.068c0-8.071,6.543-14.615, 14.615-14.615c8.072,0,14.615,6.543,14.615,14.615H43.935z"
            fill="#000"
>
            <animateTransform
                        attributeName="transform"
                        attributeType="xml"
                        dur="0.6s"
                        from="0 25 25"
                        repeatCount="indefinite"
                        to="360 25 25"
                        type="rotate"
            />
</path>,
          "height": "40px",
          "id": "loader-1",
          "style": Object {
            "enableBackground": "new 0 0 50 50",
          },
          "version": "1.1",
          "viewBox": "0 0 50 50",
          "width": "40px",
          "x": "0px",
          "xmlns": "http://www.w3.org/2000/svg",
          "y": "0px",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <animateTransform
              attributeName="transform"
              attributeType="xml"
              dur="0.6s"
              from="0 25 25"
              repeatCount="indefinite"
              to="360 25 25"
              type="rotate"
/>,
            "d": "M43.935,25.145c0-10.318-8.364-18.683-18.683-18.683c-10.318,0-18.683,8.365-18.683,18.683h4.068c0-8.071,6.543-14.615, 14.615-14.615c8.072,0,14.615,6.543,14.615,14.615H43.935z",
            "fill": "#000",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "attributeName": "transform",
              "attributeType": "xml",
              "dur": "0.6s",
              "from": "0 25 25",
              "repeatCount": "indefinite",
              "to": "360 25 25",
              "type": "rotate",
            },
            "ref": null,
            "rendered": null,
            "type": "animateTransform",
          },
          "type": "path",
        },
        "type": "svg",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
